src/avlizer_confluent.erl

```diff
@@ -254,15 +281,28 @@ decode(Bin) ->
   decode(RegId, Payload).
 
 %% @doc Decode untagged payload or with a given schema reference.
--spec decode(ref() | avro:simple_decoder(), binary()) -> avro:out().
+-spec decode(binary(), codec_options()) -> avro:out();
+            (ref(), binary()) -> avro:out();
+            (avro:simple_decoder(), binary()) -> avro:out().
+decode(Bin, CodecOptions) when is_binary(Bin) andalso is_list(CodecOptions) ->
+  {RegId, Payload} = untag_data(Bin),
+  decode(RegId, Payload, CodecOptions);
 decode(Ref, Bin) when ?IS_REF(Ref) ->
-  decode(get_decoder(Ref), Bin);
+  decode(Ref, Bin, ?DEFAULT_CODEC_OPTIONS);
 decode(Decoder, Bin) when is_function(Decoder) ->
   Decoder(Bin).
 
 %% @doc Decode avro binary with given schema name and fingerprint.
```

Comment:
Mention the new use too: decode avro binary with given schema reference and custom codec options.